crunchbase/README.md

```diff
@@ -0,0 +1,57 @@
+# Crunchbase Connector
+
+This package is a utility for connecting Cohere to Crunchbase.
```

Comment:
You can add a `Limitations` section here, where you can explain how the Crunchbase search works, eg does it support full-text search, multiple keywords, does it only search top level articles, or also comments, etc